augly/image/transforms.py

```diff
@@ -212,7 +212,6 @@ def apply_transform(
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
 
-
 class ChangeAspectRatio(BaseTransform):
```

Comment:
add newline before class definition (there should be two newlines between the return statement on 213 and the class definition on line 215)

Reply:
Done !!